sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -589,12 +601,15 @@ public partial class FileSystemInfo
     public partial class FileSystemItem
     {
         internal FileSystemItem() { }
+        public bool? IsDeleted { get { throw null; } }
         public string Name { get { throw null; } }
         public Azure.Storage.Files.DataLake.Models.FileSystemProperties Properties { get { throw null; } }
+        public string VersionId { get { throw null; } }
```

Comment:
isn't this called deletionid somewhere else? won't this be confusing with blob versioning?

Reply:
This matches `BlobContainerItem` - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs#L326